### ISSUE TITLE
orc file issue when load data to doris(version:0.12.0-rc03)

### ISSUE THREAD

**Issue by rongyousu** [apache/incubator-doris#3325]
my orc file have a column ,the type is `reg_time`   timestamp,   the value is "2000-12-07 12:09:54.0" 

my doris table column is   `reg_time` datetime 

when load the orc file to doris 's table ,the job is error :
"type:LOAD_RUN_FAIL; msg:errCode = 2, detailMessage = Parse timestamp (-416534400) error "

the be 's log :  
"W0415 19:29:22.130748 13906 orc_scanner.cpp:294] Parse timestamp (-416534400) error
W0415 19:29:22.165807 13906 broker_scan_node.cpp:423] Scanner[0] prcess failed. status=Parse timestamp (-416534400) error"

**Comment by rongyousu** [comment]
i find a same issure " Fix orc load timestamp bug #3047"  had merged into  master ,  the version:0.12.0-rc03 is new code ,but my orc file also have timestamp type ,and value like '2000-12-07 12:09:54.0'   always is error.

**Comment by HangyuanLiu** [comment]
the timestamp of '2000-12-07 12:09:54' is 976162194。Why timestamp is negative？

**Comment by rongyousu** [comment]
> the timestamp of '2000-12-07 12:09:54' is 976162194。Why timestamp is negative？
oh, i only type a example , the negtive like this: (1960-12-03 00:00:00.0 -->-286531200 )

1|-2209017600 | 1900-01-01 00:00:00.0
2 | -156844800 | 1965-01-12 00:00:00.0
3 | -286531200 | 1960-12-03 00:00:00.0
4 | -462700800 | 1955-05-05 00:00:00.0
5 | -46512000 | 1968-07-12 00:00:00.0
6 | -390211200 | 1957-08-21 00:00:00.0
7 | -235814400 | 1962-07-13 00:00:00.0
8 | -132825600 | 1965-10-17 00:00:00.0
9 | -186393600 | 1964-02-05 00:00:00.0
10 | -2180073943 | 1900-12-02 00:00:00.0
11 | -2204265943 | 1900-02-25 00:00:00.0

**Comment by HangyuanLiu** [comment]
> > the timestamp of '2000-12-07 12:09:54' is 976162194。Why timestamp is negative？
> > oh, i only type a example , the negtive like this: (1960-12-03 00:00:00.0 -->-286531200 )
> 
> 1|-2209017600 | 1900-01-01 00:00:00.0
> 2 | -156844800 | 1965-01-12 00:00:00.0
> 3 | -286531200 | 1960-12-03 00:00:00.0
> 4 | -462700800 | 1955-05-05 00:00:00.0
> 5 | -46512000 | 1968-07-12 00:00:00.0
> 6 | -390211200 | 1957-08-21 00:00:00.0
> 7 | -235814400 | 1962-07-13 00:00:00.0
> 8 | -132825600 | 1965-10-17 00:00:00.0
> 9 | -186393600 | 1964-02-05 00:00:00.0
> 10 | -2180073943 | 1900-12-02 00:00:00.0
> 11 | -2204265943 | 1900-02-25 00:00:00.0

Because the from_unixtime function currently does not support datetime earlier than 1970, it results in a time error. This problem will be solved soon, currently blocked by a pre-PR #3300

**Comment by rongyousu** [comment]
sorry ,i check the master code , the be version is  "trunk-753d6cc" ,but the issure is always there:

the be log:
W0420 19:36:44.034133 11117 orc_scanner.cpp:294] Parse timestamp (-77155200) error
W0420 19:36:44.078182 11117 broker_scan_node.cpp:423] Scanner[0] prcess failed. status=Parse timestamp (-77155200) error

**Comment by HangyuanLiu** [comment]
> sorry ,i check the master code , the be version is "trunk-753d6cc" ,but the issure is always there:
> 
> the be log:
> W0420 19:36:44.034133 11117 orc_scanner.cpp:294] Parse timestamp (-77155200) error
> W0420 19:36:44.078182 11117 broker_scan_node.cpp:423] Scanner[0] prcess failed. status=Parse timestamp (-77155200) error

This PR3300 will merge into master tomorrow

**Comment by HangyuanLiu** [comment]
#3300
